prefetch.bs

```diff
@@ -129,48 +130,73 @@ A <dfn>redirect chain</dfn> is a [=list=] of [=exchange records=].
 
 <hr>
 
-Each {{Document}} has a <dfn export>prefetch buffer</dfn>, which is a [=list=] of [=prefetch records=].
+Each {{Document}} has <dfn export for="Document">prefetch records</dfn>, which is a [=list=] of [=prefetch records=].
 
-A <dfn>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
+A <dfn export>prefetch record</dfn> is a [=struct=] with the following [=struct/items=]:
 * <dfn export for="prefetch record">URL</dfn>, a [=URL=]
 * <dfn export for="prefetch record">referrer policy</dfn>, a [=referrer policy=]
-* <dfn export for="prefetch record">sandbox flags present</dfn>, a boolean
-* <dfn export for="prefetch record">redirect chain</dfn>, a [=redirect chain=]
-* <dfn export for="prefetch record">expiry time</dfn>, a {{DOMHighResTimeStamp}}
+* <dfn export for="prefetch record">anonymization policy</dfn>, a [=prefetch IP anonymization policy=]
+* <dfn export for="prefetch record">initiator</dfn>, a {{DOMString}}
+
+  <div class="note">This is intended for use by a specification or [=implementation-defined=] feature to identify which prefetches it created. It might also associate other data with this struct.</div>
+* <dfn export for="prefetch record">state</dfn>, which is `"ongoing"` (the default), `"completed"`, or `"canceled"`
+
+  <div class="note">`"canceled"` indicates that the prefetch was aborted by the author or user, or terminated by the user agent.</div>
+* <dfn export for="prefetch record">fetch controller</dfn>, a [=fetch controller=] (a new [=fetch controller=] by default)
+* <dfn export for="prefetch record">sandboxing flag set</dfn>, a [=sandboxing flag set=]
+* <dfn export for="prefetch record">redirect chain</dfn>, a [=redirect chain=] (empty by default)
```

Comment:
I think you usually append raw responses to this, not exchange records. Not sure what the right fix is...

Reply:
I don't think so; can you point out where that happens? Initially exchange records without responses are appended using [=redirect chain/append=] and the responses are subsequently filled in with [=redirect chain/update the response=].

Reply:
I think I was reading assuming that [=redirect chain/append=] was [=list/append=].